pkg/scheduler/framework/runtime/framework.go

```diff
@@ -330,10 +330,6 @@ func NewFramework(r Registry, profile *config.KubeSchedulerProfile, opts ...Opti
 		}
 	}
 
-	if len(f.queueSortPlugins) != 1 {
```

Comment:
It seems we will not modify `queueSortPlugins` and `bindPlugins` below, so I prefer to keep this here, then we can quick return when meeting errors.

Reply:
+1 this is here for quick failures

Reply:
yes, thanks for reviews.

Reply:
Maybe simplify as 
```
if len(f.queueSortPlugins) != 1 {
    return nil, fmt.Errorf("only one queue sort plugin required for profile with scheduler name %q, but got %d", profile.SchedulerName, len(f.queueSortPlugins))
}
```

Reply:
ok, for sure. thanks quick review.